packages/btc-wallet/tsconfig.json

```diff
@@ -1,6 +1,7 @@
 {
   "compilerOptions": {
-    "baseUrl": "./"
+    "baseUrl": "./",
+    "erasableSyntaxOnly": true
```

Comment:
Can we have this defined in a "root" config where every config automatically inherits from?

If we have to create a root tsconfig and then modify every other to explicitly inherit from that one, partially defeats the purpose but ensures we don't forget to add rules to specific configs and helps adding rules to all configs at once.

Reply:
Not really easy. `extends` only supports extending from 1 config (See https://github.com/microsoft/TypeScript/issues/37884), and we're already extending from `@tsconfig/bases`, using different bases depending on the code running on the browser (the portal,btc-wallet) or not-being ui based (the rest basically)

We _could_ get away with 2 root files (one for each type) if you feel like it, but I think it increases the complexity a bit. I'm looking forward to this option being added at `@tsconfig/bases` though.

Would you still prefer me to add 2 root config files?